custom_components/intesisbox/intesisbox.py

```diff
@@ -92,10 +103,22 @@ async def query_initial_state(self):
             self._write(cmd)
             await asyncio.sleep(1)
 
+    async def delay(self, delay_time):
+        """Async Delay to slow down commands and await response from units."""
+        _LOGGER.debug(f"Delay Started for {delay_time} seconds...")
+        await asyncio.sleep(delay_time)
+        _LOGGER.debug("Delay Ended...")
+
     def _write(self, cmd):
         self._transport.write(f"{cmd}\r".encode("ascii"))
         _LOGGER.debug(f"Data sent: {cmd!r}")
 
+    async def _writeasync(self, cmd):
+        """Async write to slow down commands and await response from units."""
+        self._transport.write(f"{cmd}\r".encode("ascii"))
+        _LOGGER.debug(f"Data sent: {cmd!r}")
+        await asyncio.sleep(1)
```

Comment:
same here?

Reply:
This one gets used as '_writeasync`.  Write the command and then await the async sleep. Gets used on lines 272 and 292.  I don't know how to highlight the code like you did here.